Tsuquyomi doesn't like project paths with spaces.

I have my project in a folder like `ionic 2/newproject` which gives me the following error:
```
Tsuquyomi] Your TypeScript version is invalid. module.js:472^@    throw err;^@    ^^@^@Error: Cannot find module '/home/user/mnt/data/dev/ionic'^@    at Function
.Module._resolveFilename (module.js:470:15)^@    at Function.Module._load (module.js:418:25)^@    at Module.runMain (module.js:605:10)^@    at run (bootstrap_node.j
s:427:7)^@    at startup (bootstrap_node.js:151:9)^@    at bootstrap_node.js:542:3
```
If I rename the `ionic 2` folder to `ionic2` it works fine.

--- a/autoload/tsuquyomi/config.vim
+++ b/autoload/tsuquyomi/config.vim
@@ -108,11 +108,7 @@ function! tsuquyomi#config#tsscmd()
     if l:prj_dir !=# ''
       let l:searched_tsserver_path = s:Filepath.join(l:prj_dir, 'node_modules/typescript/bin/tsserver')
       if filereadable(l:searched_tsserver_path)
-        if !s:is_vim8
-          return g:tsuquyomi_nodejs_path.' "'.l:searched_tsserver_path.'"'
-        else
-          return g:tsuquyomi_nodejs_path.' '.l:searched_tsserver_path
-        endif
+        return g:tsuquyomi_nodejs_path.' "'.l:searched_tsserver_path.'"'
       endif
     endif
   endif

